src/cli.rs

```diff
@@ -317,6 +317,21 @@ pub struct Opt {
     #[arg(long = "detect-dark-light", value_enum, default_value_t = DetectDarkLight::default())]
     pub detect_dark_light: DetectDarkLight,
 
+    #[arg(
+        long = "diff-args",
+        short = '@',
+        default_value = "",
+        value_name = "STRING"
+    )]
+    /// Arguments to pass to `git diff` when using delta to diff two files.
+    ///
+    /// E.g. `delta --diff-args=-U999 file_1 file_2` is equivalent to
+    /// `git diff -U999 --no-index --color file_1 file_2 | delta`.
+    ///
+    /// However, if you use process substitution instead of real file paths, it falls back to `diff -u` instead of `git
+    /// diff`.
+    pub diff_args: String,
+
```

Comment:
That's actually a good use of the unique `-@` argument.

Wording: "Arguments" -> "Extra arguments",  `-U999` comes after `--color`, and give a brief example of process substitution:

... process substitution (`delta "$(grep a file_1)" "$(cmd_2)"`) instead of ...

Reply:
Thanks, done.